### ISSUE TITLE
Discuss and decide on autonomous routes

### ISSUE THREAD

**Issue by moreheadm** [FalconX-Robotics/frc2018#6]
There a bunch of different ways FalconX can approach autonomous. Should we start in the middle an go to either switch, depending on which one is ours? Should we start on on one side and go around the back if we start on the wrong side? Should we drop the cube in from the front, the side, or the back of the switch? Should we go to the scale? 

This issue is for discussion of these questions among the programming team.

**Comment by magic-robot-man** [comment]
Keep a tab on Trello: https://trello.com/c/nXuOdgZB

**Comment by moreheadm** [comment]
Yep, thanks for the link @magic-robot-man. I was thinking that Trello could be used to track progress and Github could be used for team-wide discussion. That way whenever a programming team member sees something they think we could implement or change, they can create a Github issue before the leadership decides to add it to Trello.
